examples/AN00129_hid_class/Makefile

```diff
@@ -32,6 +32,12 @@ else
 TARGET = XCORE-200-EXPLORER
 endif
 
+ifeq ($(CBUILD),1)
+EXCLUDE_FILES = main.xc \
+				endpoint0.xc
+else
+EXCLUDE_FILES = main.c
+endif
```

Comment:
if you add a XCC_FLAGS_AI_C this will make a new build config called "AI_C" .

You can then and this to the if statement for CONFIG. Something like:

ifeq ($(CONFIG),AI)
TARGET = XCORE-AI-EXPLORER
EXCLUDE_FILES = main.c
else ifeq ($(CONFIG),AI_C)
TARGET = XCORE-AI-EXPLORER
EXCLUDE_FILES = main.xc
else
 TARGET = XCORE-200-EXPLORER
EXCLUDE_FILES = main.c
 endif